static/js/admin/vue-permissions-editor/src/components/patronRoles.vue

```diff
@@ -378,7 +378,8 @@
 
                 if (users.length === 0) {
                     return users;
-                } else if (this.hasSameAssignedRoles && users[0].role === 'none' && users[1].role === 'none') {
+                } else if (this.hasSameAssignedRoles && users[0].role === 'none' && users[1].role === 'none'
```

Comment:
I'm having a hard time following this. If i'm understanding correctly, it will display a single "staff only" line if all of the following are try 1) both the assigned roles are the same, 2) the computed user roles are both "none", 3) either both the inherited roles are the same or there are no inherited roles + the assigned roles at the same.

I'm wondering why both 1 and 2 are required? Seems like the computed roles on the object would be what determines the display here, unless its to avoid rows coming and going too frequently when toggling options? Could 1 & 2 be replaced with `users[0].role === users[1].role && users[0].role === 'none'`? Guess I'm just not sure why the assigned roles matter, but I probably am missing something.

Could `hasSameInheritedRoles` return true if there are no inherited roles? That might simplify things some.